pkg/cmd/step/step_split_monorepo.go

```diff
@@ -33,17 +33,17 @@ var (
 
 	stepSplitMonorepoExample = templates.Examples(`
 		# Split the current folder up into separate Git repositories 
```

Comment:
Maybe we could expand a bit here what the command actually does. IMO that is not obvious.